app/views/pages/privacy-policy.html.haml

```diff
@@ -21,22 +21,16 @@
       %ul.govuk-list.govuk-list--bullet
         -t('static_pages.privacy_policy.what_we_collect.list').each do |li|
           %li=li.html_safe
+    %p
+      =t('static_pages.privacy_policy.what_we_collect.cookies_html')
 
     %h2.govuk-heading-m=t('static_pages.privacy_policy.how_we_use.title')
     -t('static_pages.privacy_policy.how_we_use.text').each do |p|
       %p=p.html_safe
 
     %h2.govuk-heading-m=t('static_pages.privacy_policy.why_its_lawful.title')
-    %p=t('static_pages.privacy_policy.why_its_lawful.about')
-
-    %h2.govuk-heading-m=t('static_pages.privacy_policy.who_we_make_available_to.title')
-    %p=t('static_pages.privacy_policy.who_we_make_available_to.line_1')
-    %p=t('static_pages.privacy_policy.who_we_make_available_to.line_2')
-    %p
-      =t('static_pages.privacy_policy.who_we_make_available_to.line_3')
-      %ul.govuk-list.govuk-list--bullet
-        -t('static_pages.privacy_policy.who_we_make_available_to.list').each do |li|
-          %li=li.html_safe
+    -t('static_pages.privacy_policy.why_its_lawful.about').each do |p|
+      %p=p.html_safe
```

Comment:
In general - though not in this old file - our style has been to put a space after the `=`. Could you find-and-replace `=t` to `= t`, and also for `=p` and `=li`.